package.json

```diff
@@ -8,53 +8,40 @@
   "author": "",
   "license": "LGPL-2.1",
   "scripts": {
-    "build": "webpack",
-    "watch": "webpack --watch",
-    "clean": "cd ./dist && rm -rf *",
-    "eslint": "eslint --ext .jsx --ext .js src/ lib/",
-    "eslint:fix": "eslint --fix --ext .jsx --ext .js src/ lib/"
+    "build": "./build.js",
+    "watch": "./build.js -w",
+    "eslint": "eslint --ext .jsx --ext .js src/",
+    "eslint:fix": "eslint --fix --ext .jsx --ext .js src/"
   },
   "devDependencies": {
-    "@babel/core": "^7.6.0",
-    "@babel/eslint-parser": "^7.13.14",
-    "@babel/preset-env": "7.12.17",
-    "@babel/preset-react": "^7.0.0",
     "argparse": "^2.0.1",
-    "babel-loader": "^8.0.0",
-    "chrome-remote-interface": "^0.32.1",
-    "compression-webpack-plugin": "^9.0.0",
-    "copy-webpack-plugin": "^9.0.0",
-    "css-loader": "^6.5.0",
-    "css-minimizer-webpack-plugin": "^3.0.1",
-    "eslint": "^7.10.0",
-    "eslint-config-standard": "^14.1.1",
-    "eslint-config-standard-jsx": "^10.0.0",
-    "eslint-config-standard-react": "^11.0.0",
-    "eslint-plugin-flowtype": "^5.2.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsx-a11y": "^6.3.1",
```

Comment:
Note that this whole change sneakily changed the pinning of deps from allowing not allow auto updating [minor/patch level changes](https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004) anymore.

Reply:
argh, yes I'll add the ^ back